example/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "example",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
Looks like `prettier-package-json` isn't formatting this file. Also, are all these `dependencies` or are there some `devDependencies`?

Reply:
I've now included `prettier-package-json` in a standard `format` command.

> Also, are all these dependencies or are there some devDependencies?

These are all technically devDependencies, but this matches the default style of `create-react-app`. It also maintains an open version carrot `^`.